girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -496,7 +536,8 @@ class AnnotationSchema:
                 'title': 'Image Markup',
                 'description': 'Subjective things that apply to a '
                                'spatial region.'
-            }
+            },
+            'overlay': overlaySchema
```

Comment:
I'd rather this was just another possible annotation element.  We can expect the viewer to do something similar to how we handle heatmaps -- in `web_client/annotations/convertFeatures.js` we create a layer per heatmap (which isn't actually strictly necessary, as two heatmaps could share a layer) and manage it there.

Reply:
See d0aa882. Once I added image overlays as an option for elements, I was hitting errors on upload of a new annotation during the calculation of the bounding box.

My naive approach to bounding box calculation for image overlays was just to set it to the size of the entire base image. Perhaps a better approach would be to use the transform properties to calculate the exact coordinates of the overlay. Thoughts?

Reply:
It would be better to calculate the actual bounding box, since, in principle, you could have several images and only need to fetch the ones that intersect some region.  Similarly, the overlay could actually extend outside of the base image.

Reply:
I just committed 9c44499, which should do a better job of calculating the bounding box. Let me know if you have any additional thoughts/suggestions.